packages/bitcore-node/scripts/migrateTestnetWallets.js

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env node
+
+const { WalletStorage } = require('../build/src/models/wallet');
+const { WalletAddressStorage } = require('../build/src/models/walletAddress');
+const { Storage } = require('../build/src/services/storage');
+const { wait } = require('../build/src/utils');
+
+const chain = 'BTC';
+const oldNetwork = 'testnet3';
+const newNetwork = 'testnet4';
+const dryRun = true; // make sure you stop sync services before running this script
+const batchSize = 10000;
+
+let quit = false;
+process.on('SIGINT', () => {
+  if (quit) {
+    process.exit(1);
+  }
+  console.log('Caught interrupt signal');
+  quit = true;
+});
+
+Storage.start()
+  .then(async () => {
+    console.log('Connected to the database');
+
+    const cnt1 = await WalletStorage.collection.countDocuments({ chain, network: oldNetwork });
+    const cnt2 = await WalletAddressStorage.collection.countDocuments({ chain, network: oldNetwork });
+
+    console.log(`${chain} ${oldNetwork} wallets:`, cnt1);
+    console.log(`${chain} ${oldNetwork} walletAddresses:`, cnt2);
+
+    if (dryRun) {
+      return;
+    }
+    console.log('----------------------------------------');
+
+    console.log(`Updating ${chain} ${oldNetwork} => ${newNetwork} wallets in 10 seconds...`);
+    !quit && await wait(10000);
+    console.log(`Updating ${chain} ${oldNetwork} wallets...`);
+    let wallets = await WalletStorage.collection.find({ chain, network: oldNetwork }).project({ _id: 1 }).limit(batchSize).toArray();
+    let walletsUpdated = 0;
```

Comment:
This (and walletAddressesUpdated) is assigned & modified - but never used.

Reply:
It is used in the log inside the while loop to output the % completed